contracts/get_contract_metadata/assembly/__tests__/Getcontractmetadata.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('Get Contract Metadata', () => {
```

Comment:
Needs testing

Reply:
Appears as tests are in a different source file. If this file is not needed, let's delete it.

Reply:
Correct, I'm using `local Koinos` which is a package I created to do integration tests using Typescript. Unit tests in AS are great but are hard to build for integration tests.